x-pack/plugins/security_solution/cypress/integration/detection_rules/bulk_edit_rules.spec.ts

```diff
@@ -119,6 +119,7 @@ describe('Detection rules, bulk edit', () => {
 
     // check if rule has been updated
     cy.get(CUSTOM_RULES_BTN).click();
+    cy.wait(1000);
```

Comment:
Is this just to address some flake? If waiting on content to interact with like `goToTheRuleDetailsOf` is doing, it'd be best to add a `timeout` to that `get` instead of an arbitrary wait, that way we're not waiting more than we need.


Though would be good to double-check if this is needed as there's [another open PR](https://github.com/elastic/kibana/pull/135208/files#diff-b71b427ad3513e7c406ac1624899a3084dfdb1246fe3b9dfc09474920a2390dc) that doesn't seem to be having issues with this test?

Reply:
Indeed it's to address the flake, but this flake tend to fail all the time for me... ;( There is click with force:true in there, which might be causing the issue. I didn't want to get too involved in your tests, so decided just to ''skip'' the problem. However, if you're familiar with the issue, or prefer to not use cy.wait (:P) I'll be happy to help.

Reply:
Ahh, thank you for verifying! Looks like this has since been addressed in https://github.com/elastic/kibana/pull/135280, so we should be good once this branch is on the latest from `main`.